driver-core/src/test/functional/com/mongodb/client/test/CollectionHelper.java

```diff
@@ -380,6 +380,18 @@ public void killAllSessions() {
         }
     }
 
+    public void renameCollection() {
```

Comment:
This is too specific.  It should rename the collection that this is a helper for (see the namespace property) to a name that is passed as a parameter, rather than using hard-coded `from` and `to`.

Reply:
Done